docker-entrypoint.sh

```diff
@@ -118,6 +125,13 @@ if [[ ! -f /db/init_done ]]; then
 			exit 1
 		fi
 	fi
+
+	if [[ "${OVERPASS_STOP_AFTER_INIT}" == "false" ]]; then
+		echo "Overpass container ready to receive requests"
+	else
+		echo "Overpass container initialization complete. Exiting."
+		exit 0
+	fi
```

Comment:
Can we just move this one `fi` below, into the main, just before `envsubst` is called, and this will simplify the whole script, as we will need to check only once?